mlflow/models/evaluation/base.py

```diff
@@ -481,6 +490,10 @@ def evaluate(
         :param run_id: The ID of the MLflow Run to which to log results.
         :param evaluator_config: A dictionary of additional configurations for
                                  the evaluator.
+                                 evaluator_config may contain
+                                 - is_baseline_model: A boolean indicating whether the evaulation
+                                   is for baseline model. For baseline model, no artifact will
+                                   be generated and no metric will be logged.
```

Comment:
This seems to imply that we intend to have `mlflow.evaluate()` call `default_evaluator.evaluate()` twice; once with `is_baseline_model=False` and once with `is_baseline_model=True`. I don't think we should be doing this. Instead, we should make a single call to `default_evaluator.evaluate()` and pass in the baseline model as its own argument (e.g. `baseline_model`).

Reply:
Made changes as directed, `ModelEvaluator.evaluate` now takes optional param `baseline_model` and return tuple of `EvaluationResult`